src/core/client/stream/tabs/Comments/Comment/ReplyCommentForm/ReplyCommentFormContainer.tsx

```diff
@@ -186,10 +202,58 @@ export class ReplyCommentFormContainer extends Component<Props, State> {
     }
   };
 
+  private jumpToComment = (commentID: string | null, onClose?: () => void) => {
+    if (!commentID) {
+      return;
+    }
+
+    if (onClose) {
+      onClose();
+    }
+
+    const elementID = `comment-${commentID}`;
+    setTimeout(() => {
+      const element = document.getElementById(elementID);
+      if (element) {
+        element.scrollIntoView({ behavior: "smooth", block: "start" });
+      }
+    }, 300);
+  };
+
   public render() {
     if (!this.state.initialized) {
       return null;
     }
+
+    if (this.state.showJumpToComment) {
+      return (
+        <CallOut
+          title={
+            <Localized id="comments-jumpToComment-title">
+              Your reply has posted below
+            </Localized>
+          }
+          icon={<Icon>question_answer</Icon>}
+          iconColor="none"
+          color="primary"
+        >
+          <Localized id="comments-jumpToComment-GoToReply">
+            <Button
+              onClick={() => {
+                this.jumpToComment(
+                  this.state.jumpToCommentID,
+                  this.props.onClose
+                );
+              }}
```

Comment:
As `jumpToComment` is already inside the component, you don't need to pass these parameters to it. If you refactored it to instead receive nothing, you can rewrite this as:

```suggestion
              onClick={() => {
                this.jumpToComment();
              }}
```